JSDemos/Demos/Toast/Overview/React/ProductItem.js

```diff
@@ -5,7 +5,7 @@ import { CheckBox } from 'devextreme-react/check-box';
 export function ProductItem(props) {
   const onValueChanged = React.useCallback((e) => {
     props.checkAvailability(e, props.product);
-  }, []);
+  }, [props]);
```

Comment:
is there any point in this dep/useCallback? because everytime this component rerenders (no state, just props) onValueChanged will be a new function

Reply:
I've added the deps because it is lint requirement